libs/ml/include/ml/ops/estimation/charge_constants.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2020 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "meta/vm_types.hpp"
+
+namespace fetch {
+namespace ml {
+namespace ops {
+namespace charge_cost {
+
+using fetch::vm::ChargeAmount;
+
+static constexpr ChargeAmount DROPOUT_PER_ELEMENT             = 3;
```

Comment:
where do these numbers come from by the way?

Reply:
for now these constants are just dummies to be used in each Ops to return their charge, I will update the OR to show how are they used